File: 3 КУРС/2 семестр/Учебная практика/JavaProects/app/src/main/java/com/example/voiceassistent/Number.java
package com.example.voiceassistent;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class Number implements Serializable {

    @SerializedName("str")
    @Expose
    public String number;

}
//https://htmlweb.ru/json/convert/num2str?num=34

File: 3 КУРС/2 семестр/Учебная практика/kotlin/kotlin/app/src/main/java/com/example/kotlinapp/NumberConversionAPI/Number.kt
package com.example.kotlinapp.NumberConversionAPI

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable


class Number : Serializable {
    @SerializedName("str")
    @Expose
    var str: String? = null
}